bullet_train-api/lib/bullet_train/api/version.rb

```diff
@@ -1,5 +1,5 @@
 module BulletTrain
   module Api
-    VERSION = "1.3.22"
+    VERSION = "1.3.23"
```

Comment:
@abradburne, something is weird with this PR (and with #522 now). All of these version changes shouldn't be here. I'm not sure what would lead to that, but can you try rebasing against `main` again? If that doesn't do the trick let me know and we'll figure something out.

Reply:
um, weird. Let me take a look.